api/graphql/makeModels.js

```diff
@@ -42,8 +42,19 @@ export default function makeModels (userId, isAdmin) {
     users: {
       typename: 'Person',
       model: User,
-      attributes: ['id', 'name', 'avatar_url', 'banner_url'],
-      relations: ['posts'],
+      attributes: [
+        'id',
+        'name',
+        'avatar_url',
+        'banner_url',
+        'bio',
+        'twitter_name',
+        'linkedin_url',
+        'facebook_url',
+        'url',
+        'location'
+      ],
+      relations: ['memberships', 'posts'],
```

Comment:
You don't want to add the filter to `users`, but to `communities_users`. It's access to the latter that we want to control.

Btw, n.b., I have changes to the same files in #242. Feel free to review that one and I'll merge it.